electron-core/src/electron/CapacitorElectronApp.ts

```diff
@@ -268,7 +268,9 @@ export class CapacitorElectronApp {
       ) {
         this.splashScreenReference.hide();
       } else {
-        this.mainWindowReference.show();
+        if (browserWindowConfig.show) {
```

Comment:
@IT-MikeS actually this is the only line what matters to me. I need a way to set `show: false` in the configuration stage and never show the window on startup, is there another way to achieve that?

Reply:
Your goal is to both not use the splash screening not have the window display at first until you specifically tell it to?

Reply:
@IT-MikeS yeah my use case is centred in the system tray, so I don't need to show the window/splashscreen on startup. although I need its rendering process, so my plan is to start the app with the window hidden and control it's exhibition later

Reply:
I managed to do it with this PR

Reply:
Okay I understand, I will look at it tomorrow when I can just to make sure it's as clean as possible but I think you've got it 👍

Reply:
Thank you @IT-MikeS